lib/termination-poller.js

```diff
@@ -71,7 +76,7 @@ class TerminationPoller {
           InstanceIds: instanceIds,
         });
       } catch (err) {
-        log.warn({err}, 'failed to describe instances in api');
+        log.warn({err}, 'failed to describe instances for termination');
```

Comment:
It looks like this and a few other changes in this file should be cherry-picked from 93afe77 to 321c4c1